app/assets/javascripts/widgets/wysiwyg.js

```diff
@@ -0,0 +1,14 @@
+$(document).ready(function(){
+  var anchorLinks = $('h3.anchorLinks a, a[href="#top"]');
+  var navHeight = $("#cu_nav").outerHeight();
+
+  /* offset omninav when interal anchor links scroll to target */
+  anchorLinks.on('click', function(e){
+    var target = $('[name=' + this.hash.slice(1) + ']');
```

Comment:
The `event` object is referring to the link that was clicked. This selector is finding the page location to scroll to, which is identified by the name attribute.